README.md

```diff
@@ -44,7 +44,25 @@ $ slack create my-app -t slack-samples/deno-starter-template
 $ cd my-app
 ```
 
-## Create a Link Trigger
+## Running Your Project Locally
+
+While building your app, you can see your changes propagated to your workspace
+in real-time with `slack run`. You'll know an app is the development version if
+the name has the string `(local)` appended.
+
+```zsh
+# Run app locally
+$ slack run
+
+Connected, awaiting events
+```
+
+To stop running locally, press `<CTRL> + C` to end the process.
+
+## Creating Triggers
+
+When you run or deploy your project, the CLI will prompt you to create a new
```

Comment:
Maybe to make it slightly clearer / push to action, we monospace the two CLI action words?

```suggestion
When you `run` or `deploy` your project, the CLI will prompt you to create a new
```